suites/pacific/rbd/tier-1_rbd_mirror.yaml

```diff
@@ -166,7 +166,7 @@ tests:
             imagesize: 2G
             io-total: 200M
             resize_to: 5G
-      polarion-id: CEPH-83573332
+      polarion-id: CEPH-83573332, CEPH-9511
```

Comment:
@rkhambad Is it ok if i add those steps in the existing polarion id and make 9511 as Inactive?

Reply:
@Manohar-Murthy @VasishtaShastry To avoid multiple polarion ID, can we make `CEPH-9511` as Inactive by adding this test case step in `CEPH-83573332`?
Please let me know your thoughts ?
creating a separate test module for this negative scenario is redundant, in this existing mirror setup 
we can cross check this negative scenario

Reply:
@sunilangadi2 
IMO we can avoid above mentioned TC to be merged
we should block the build if CEPH-83573332 fails but not if CEPH-9511
I see convinience as the only reason being mentioned here to merge, which I'm not able to agree with

Reply:
@VasishtaShastry Agree with your point, but when test case fails or blocking build once again we will cross check manually Is it at that time can we decide is it?
My only concern is, to cover this negative scenario, we just have repeat some of the existing steps
I though of avoiding
Please let me know your thoughts once again?
i am ok to go with new module.

Reply:
In general,
-> but when test case fails or blocking build once again we will cross check manually Is it at that time can we decide is it?
cross checking is done on rotation basis, new person would be checking them periodically, i don't see any validation to create this dependency between monitor and SMEs&authors each time.

-> I  think it should be okay if we merge similar/overlapping TCs of same tier

Reply:
ok sure let me cross check then.

Reply:
Please have two seperate tests as scenarios are different. Develop another module for negative scenario.

Reply:
Done added new module itself.